### PR TITLE
Feature/new config file

### PR DESCRIPTION
**Pull request by dsementsov**

### CODE REVIEW

File: coc-dashboard/model/database.py [hunk 1]
```diff
@@ -1,66 +1,27 @@
...
-            "indicator_id": self.id,
-            "indicator_group": self.group,
-            "indicator_name": self.name,
-            "indicator_view": self.view,
+            "config_indicator": self.indicator,
+            "config_function": self.function,
+            "config_group": self.group,
+            "config_view": self.view,
+            "config_denominator": self.denominator,
         }
```
Comment (dsementsov) [inline coc-dashboard/model/database.py:26]: @property
def is_ratio(self):
    return if self.function == "ratio"

File: coc-dashboard/store/database.py [hunk 2]
```diff
@@ -110,22 +114,13 @@ def get_repository(self, repo_name):
                 pass
```
Comment (dsementsov) [inline coc-dashboard/store/database.py:109]: __dataframe["date"] = __dataframe["date"].astype("datetime64[ns]")

File: coc-dashboard/store/database.py [hunk 3]
```diff
@@ -163,44 +158,84 @@ def fetch_dataset(self, key):
...
     def filter_by_indicator(self, df, indicator):
-        df = df.reset_index()
-        try:
-            df = df[self.index_columns + [indicator]]
-        except Exception as e:
-            print(e)
-            print("No such column is present in the dataframe")
+
+        config = self.get_serialized_into_df(Config)
+
```
Comment (dsementsov) [inline coc-dashboard/store/database.py:165]: session = self.get_session()
indicator_config = session.query(Config).filter_by(name=indicator).one()

if indicator_config.is_ratio:

Comment (dsementsov) [inline coc-dashboard/store/database.py:165]: pd.DataFrame(self.__inidcator_serialized)

**Comment by nbigaud** [comment]
Closed this pull request since I have a new one coming now

**Pull request closed without merging**